Javascript/dom-utils.js

```diff
@@ -0,0 +1,14 @@
+let resetButton = document.querySelector("#reset-button");
+let inputEnglish = document.querySelector("#textarea__input--english");
+let inputMorse = document.querySelector("#textarea__input--morse");
+let labelInput = document.querySelector("#button__english--submit");
+let labelMorse = document.querySelector("#button__morse--submit");
+
+//Morse Buttons
+let buttonsForMorce = document.getElementById("container__buttons-for-morse");
+var morsebtns = document
+  .querySelector(".container__buttons-for-morse")
+  .querySelectorAll("button");
```

Comment:
Why is this a var?

Reply:
Haven't really looked into it before. Its a mistake. I am trying to use let for all the variables. Changed that one as well